openmpi.spec

```
@@ -5,6 +5,7 @@
 %define branch v5.0.x
 %define tag v%{realversion}
 Source: git+https://github.com/open-mpi/ompi.git?obj=%{branch}/%{tag}&export=%{n}-%{realversion}&submodules=1&output=/%{n}-%{realversion}.tgz
+Source1: https://github.com/open-mpi/ompi/commit/aa5577441ff1ab7f97f8b63e442b37457c7bd997.diff
```

```suggestion
Source1: https://github.com/open-mpi/ompi/commit/15d97a054a286c7bf8e4296f6fed2ca6cd43f063.diff
```

lets use the commit from https://github.com/open-mpi/ompi/compare/v5.0.x branch

---

FYI @fwyzard , the change from v5.0.x branch works. Can you please push this change ?